src/Tickets/Commerce/Shortcodes/Checkout_Shortcode.php

```diff
@@ -108,17 +101,17 @@ public function get_gateways_active() {
 	}
 
 	/**
-	 * Get the number of connected gateways.
+	 * Get the number of available gateways that can be used for checkout. A gateway is only available if it's properly connected and enabled.
 	 *
 	 * @since 5.2.0
 	 *
-	 * @return int The number of connected gateways.
+	 * @return int The number of available gateways.
 	 */
-	public function get_gateways_connected() {
+	public function get_gateways_available() {
```

Comment:
I am not sure if we should get rid of a public method that was in place, as it can be already being used by other people. Maybe we should create a new one for available and keep having the old one in place.